ctapipe/image/waveform_cleaning.py

```diff
@@ -78,7 +78,8 @@ class BaselineWaveformCleaner(WaveformCleaner):
 
     def apply(self, waveforms):
         # Subtract baseline
-        baseline_sub = waveforms - np.mean(waveforms[:, :, self.sample_start:self.sample_end],
+        baseline_sub = waveforms - \
```

Comment:
Please don't use explicit line contiuations with `\`, use parenthese.

Maybe even split like this:
```
baseline = np.mean(
    waveforms...,
)

return waveforms - baseline```